templates/shared/user/profile_big_avatar.tmpl

```diff
@@ -1,4 +1,4 @@
-<div id="profile-avatar-card" class="ui card">
+<div id="profile-avatar-card" class="ui card tw-rounded">
```

Comment:
I have seen not only one `ui card tw-rounded`. 

Should it write border styles into the `.ui.card` selectors?

Reply:
Hmm yes, I think we could.

Reply:
Fixed in https://github.com/go-gitea/gitea/pull/30707/commits/2bd28a7f73ebac4f56af5d65ef6bca8e10ad2029.